version: '3'
services:
  spin-mdaca-db:
    image: mysql:8.0
    container_name: spin-mdaca-db
    restart: on-failure
    ports:
      - "3306:3306"
    volumes:
      - ./mysql/data:/var/lib/mysql
      - ./init-chats.sql:/docker-entrypoint-initdb.d/init-chats.sql
      - ./init-threads.sql:/docker-entrypoint-initdb.d/init-threads.sql
    environment:
      MYSQL_DATABASE: ${DB_NAME}
      MYSQL_USER:     ${DB_USER}
      MYSQL_PASSWORD: ${DB_PASS}
    healthcheck:
      test: ["CMD", "mysqladmin" ,"ping", "-h", "localhost"]
      timeout: 20s
      retries: 10
    networks:
      - mdaca-cse-ami-network
networks:
  mdaca-cse-ami-network:
    external: true
  